fdbserver/masterserver.actor.cpp

```diff
@@ -665,7 +669,12 @@ ACTOR Future<Void> readTransactionSystemState( Reference<MasterData> self, Refer
 	self->configuration.fromKeyValues( rawConf );
 	self->originalConfiguration = self->configuration;
 	self->hasConfiguration = true;
-	TraceEvent("MasterRecoveredConfig", self->dbgid).detail("Conf", self->configuration.toString()).trackLatest("RecoveredConfig");
+
+	TraceEvent("MasterRecoveredConfig", self->dbgid)
+		.setMaxEventLength(11000)
```

Comment:
I'm not sure what the general convention of using magic numbers are in FDB, but should this be defined as separate constant somewhere? Also, since this is relatively rare log as it happens during recruitment, why not just use same values (11000) for both event and field?

Reply:
To me these constants seemed pretty similar to adding a suppression to trace events, for which we use hard-coded constants. I'm not married to that idea, though, if someone has strong feelings that we should extract the constant. I will acknowledge that the number chosen for the events modified here are correlated more than a suppression value likely would be.

As for using 11000 for the event length, I typically set the event length longer than the field that I'm extending. This is because the field length only restricts the size of the value for the fields it affects, while the event length includes key and value sizes for all fields. Maybe it's something we should change, but currently overflowing a field is handled more gracefully than overflowing an event, so doing it this way allows us to better handle a value that manages to exceed the field limit.